src/metabase/premium_features/settings.clj

```diff
@@ -250,8 +250,10 @@
   "Should Metabase generate SQL queries?"
   :ai-sql-generation)
 
-(define-premium-feature ^{:added "0.55.0"} enable-embedding-iframe-sdk?
-  "Should we allow users to embed the SDK in iframes?"
+; the "-feature" suffix avoids name collision with the setting getter
+(define-premium-feature ^{:added "0.55.0"} enable-embedding-simple-feature?
+  "Should we enable simple embedding features?"
+  ; TODO: rename the feature token name to `:embedding-simple` before beta!
   :embedding-iframe-sdk)
```

Comment:
TODO: rename premium feature name to embedding-simple (to check with @albertoperdomo first because we need to update Harbormaster)